scripts/lldpshow

```diff
@@ -134,11 +134,16 @@ class Lldpshow(object):
                     continue
                 remote_port = intf.find('port')
                 r_portid = remote_port.find('id').text
-                key = l_intf + "#" + r_portid
+                chassis = intf.find('chassis')
+                rmt_name = chassis.find('name')
+                if rmt_name is not None: 
+                    rmt_host_name = rmt_name.text
+                else:
+                    rmt_host_name = ""
+                key = l_intf + "#" + rmt_host_name + "#" + r_portid
```

Comment:
This key is split with '#' later on eg: https://github.com/sonic-net/sonic-utilities/blob/c699b0404985b2371b1273ae397d7ca4d67c8158/scripts/lldpshow#L164
Need to check if this change affects that.

Reply:
I checked it before I made the change.  It only takes the first filed after split.  The modification won't impact it.